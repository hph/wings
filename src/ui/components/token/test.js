import React from 'react';
import renderer from 'react-test-renderer';

import Token from './index';

describe('Token', () => {
  it('should render children in a span', () => {
    expect(renderer.create(<Token>token</Token>).toJSON()).toMatchSnapshot();
  });

  it('should set a className prop based on the type', () => {
    expect(
      renderer.create(<Token type="comment">token</Token>).toJSON(),
    ).toMatchSnapshot();
  });
});
