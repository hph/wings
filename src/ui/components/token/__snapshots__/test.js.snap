// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Token should render children in a span 1`] = `
<span
  className="undefined"
>
  token
</span>
`;

exports[`Token should set a className prop based on the type 1`] = `
<span
  className="comment"
>
  token
</span>
`;
